R/bibstats.R

```diff
@@ -0,0 +1,126 @@
+#' Reporting bibliographic statistics
+#' 
+#' These functions are useful for calculating salient statistics
+#' on the bibliographies used for dissertations or syllabi.
```

Comment:
Add more description. Also add examples?

Reply:
The description has been improved. Since the functions require external files, we'd have to put in non-running examples with dummy paths. Should I include them or is the description sufficient?

Reply:
Hmm, ok, no examples for now. How have the descriptions been improved?

Reply:
Okay, for the examples. About the description, I described the functions' purpose individually in a list